crates/weaver_forge/src/extensions/otel.rs

```diff
@@ -24,6 +25,7 @@ pub(crate) fn add_tests_and_filters(env: &mut minijinja::Environment<'_>) {
     env.add_filter("not_required", not_required);
     env.add_filter("instantiated_type", instantiated_type);
     env.add_filter("enum_type", enum_type);
+    env.add_filter("semconv_const", semconv_const);
```

Comment:
since most of the cases don't make sense in the context of `semconv_const` (and we only need it as a conveniece), I'd prefer to have a filter per class of constants:

- `pascal_case_const`
- `camel_case_const`
- snake_case_const`
- `screaming_snake_case_const`

That's all we have in build-tools and it seems it was sufficient for all languages.

Reply:
they are called differently: https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/src/opentelemetry/semconv/templating/code.py#L166-L181

but my main point is that `semconv_const(name, "title")` does not make sense, so instead of allowing someone to use it wrong with arbitrary casing, we'd rather provide a specific method.

Reply:
Changed the code to follow your proposal.